app-api/src/main/java/org/openmrs/eip/app/management/entity/SenderSyncResponse.java

```diff
@@ -21,6 +21,9 @@ public class SenderSyncResponse extends AbstractEntity {
 	@Column(name = "date_sent_by_receiver", nullable = false, updatable = false)
 	private LocalDateTime dateSentByReceiver;
 	
+	@Column(name = "receiver_date_received")
```

Comment:
This column must be required

Reply:
can't be required because of the syncResponseModel event that does not have the value.

Reply:
Why would it not have a value when you are clearly setting in the ReceiverActiveMqMessagePublisher?

Reply:
Sorry, I thought a syncReponseModel was currently used in production so I didn't put the attribute as required.